lib/soapy_cake/admin_batched.rb

```diff
@@ -28,7 +28,9 @@ def initialize(admin, method, opts, limit)
       def to_enum
         Enumerator.new do |y|
           loop do
-            result = admin.public_send(method, opts.merge(row_limit: limit, start_at_row: offset))
+            result = admin
+              .public_send(method, opts.merge(row_limit: limit, start_at_row: offset))
+              .to_enum
```

Comment:
I don't think we need these benefits for mocked data. 1.11.5 (a minor version bump after all) broke the tests. I only revert to the old behavior.